designsafe/static/scripts/ng-designsafe/services/file-operation-service.js

```diff
@@ -50,9 +50,7 @@ export class FileOperationService {
     getTests(files) {
         const externalDataStates = ['boxData', 'dropboxData', 'googledriveData'];
         const agaveDataStates = ['myData', 'projects.view', 'projects.curation'];
-        let isHazmapper = files.length > 0
-            ? files.some(e => e.name.endsWith('hazmapper'))
-            : false;
+        let isHazmapper = files.length > 0 ? files.some((e) => e.name.endsWith('hazmapper')) : false;
```

Comment:
I love this, can you leave a quick comment on this line as to what it's supposed to do for future devs' readability?